airbyte-webapp/src/components/EntityTable/components/ChangesStatusIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classnames from "classnames";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { Tooltip } from "components/ui/Tooltip";
+
+import { SchemaChange } from "core/request/AirbyteClient";
+import { convertSnakeToCamel } from "utils/strings";
+
+import styles from "./ChangesStatusIcon.module.scss";
+
+interface ChangesStatusIconProps {
+  schemaChange?: SchemaChange;
+}
+
+export const ChangesStatusIcon: React.FC<ChangesStatusIconProps> = ({ schemaChange = "no_change" }) => {
+  if (schemaChange === "no_change") {
+    return null;
+  }
+  const iconStyle = classnames(styles.changesIcon, {
+    [styles.breaking]: schemaChange === "breaking",
+    [styles.nonBreaking]: schemaChange === "non_breaking",
+  });
+  return (
+    <Tooltip
+      containerClassName={styles.tooltipContainer}
+      control={<FontAwesomeIcon className={iconStyle} icon={faExclamationCircle} size="2x" />}
```

Comment:
Add `placement="left"` Otherwise, the tooltip doesn't look great for breaking changes:
![image](https://user-images.githubusercontent.com/168664/205389935-fe683847-7051-4f44-8f94-d7ae088dba94.png)


With `placement="left"`:
![image](https://user-images.githubusercontent.com/168664/205390684-110c5afa-961d-47f2-876a-b9e881483d1d.png)